index.html

```diff
@@ -940,19 +945,37 @@
 
 ### Rights {#data-rights}
 
-To the extent possible, [=people=] should retain a certain number of rights over their data, and
-these rights should be facilitated by the [=parties=]. While data rights are a self-governing
-mechanism that is not sufficient to support [=privacy=], they do complement self-determination and
-help improve accountability. Such rights should include:
+[=People=] retain certain rights over data about themselves, and these rights should be facilitated by their [=user agent=] and the [=parties=]. While data rights alone are not sufficient to support [=privacy=], they do support self-determination and help improve accountability. Such rights include:
 
 * The right to <em>access</em> [=data=] about oneself.
-* The right to <em>erase</em> [=data=] about oneself, with or without terminating the service (though
-  what [=data=] can be erased may differ between those two cases).
-* The right to <em>port</em> information that one has stored with a given [=party=] in such a way
-  that it can easily be reused or transferred elsewhere.
+
+  This right includes being able to discover data about oneself (no secret databases) and to review what information has been collected or inferred.
+
+* The right to <em>erase</em> [=data=] about oneself, sometimes described as a <em>right to be forgotten</em>. 
+
+  The right to erase applies whether or not terminating use of a service altogether, though what [=data=] can be erased may differ between those two cases. On the Web, [=people=] may wish to erase data on their device, on a server, or both, and the distinctions may not always be clear.
+
+* The right to <em>port</em> [=data=], including data one has stored with a [=party=], so it can easily be reused or transferred elsewhere.
+
+  Portability is needed to realize the ability for [=people=] to make choices about services with different data practices. Standards for interoperability are essential for effective re-use.
```

Comment:
I'm not sure data that's stored locally interacts with privacy rights?

```suggestion
* The right to <em>port</em> [=data=] from one steward to another, so [=people=] can make real choices between services with different data practices. Standards for interoperability are essential for effective re-use.
```

Reply:
I think locally stored data definitely has privacy impacts, and that it may support this right to portability to include data stored by the user agent or in local storage for a site. 

"steward" isn't a defined term or used elsewhere in the doc yet, but if that is a common concept, it could certainly apply here (e.g. a term for "any parties the user is interacting with and the user agent").

Reply:
I agree that locally stored data has privacy impacts, you could do all kinds of bad things to people using data that's just on the client. It's also important for portability (as Wordle users would agree).

Data steward is an existing concept but I haven't seen it used to describe the company that manages data, only the people inside a company who do (which doesn't mean that it's not potentially a good use). At The Times, "data stewards" are the people who get ~radicalised~ trained in privacy and algo ethics across all teams of the company and know what questions to ask, what to worry about, who to talk to, etc. It's short of a fiduciary but better than some rando who might have your data. I'd be curious to hear about other uses.